app/src/main/java/com/a0xffffffff/dinesum/RequestTracker.java

```diff
@@ -4,11 +4,37 @@
 
 public class RequestTracker {
 
+    private static RequestTracker mRequestTracker = new RequestTracker();
+
     private User mCurrentUser;
     private Request[] mNearbyRequests;
 
     private RequestTracker() {
+        mRequestTracker = RequestTracker.getInstance();
```

Comment:
If we're doing eager initialization, I think the private constructor could just be empty? 
I'd imagine this would throw an NPE when testing because mRequestTracker isn't initialized yet when getInstance() is called

Reply:
Good catch, just removed it!